Makefile

```diff
@@ -595,6 +595,7 @@ TESTS = \
 	db_secondary_test \
 	block_cache_tracer_test \
 	block_cache_trace_analyzer_test \
+	defer_test \
```

Comment:
Nit: you also need to update src.mk and  CMakelists.txt. Then run `python buckifier/buckify_rocksdb.py` in order to update TARGETS file.

Reply:
updated, thanks.